app/components/frame-viewer.cjsx

```diff
@@ -86,9 +104,34 @@ module.exports = React.createClass
         </div>
 
     if FrameWrapper
-      <FrameWrapper frame={frame} naturalWidth={@state.frameDimensions?.width or 0} naturalHeight={@state.frameDimensions?.height or 0} workflow={@props.workflow} subject={@props.subject} classification={@props.classification} annotation={@props.annotation} loading={@state.loading} onChange={@props.onChange}>
-        {frameDisplay}
-      </FrameWrapper>
+      <div>
+        <FrameWrapper frame={frame} naturalWidth={@state.frameDimensions?.width or 0} naturalHeight={@state.frameDimensions?.height or 0} panByDrag={@panByDrag} viewBoxDimensions={@state.viewBoxDimensions or "0 0 0 0"} workflow={@props.workflow} subject={@props.subject} classification={@props.classification} annotation={@props.annotation} loading={@state.loading} onChange={@props.onChange} panEnabled={@state.panEnabled} >
+          {frameDisplay}
+        </FrameWrapper>
+        {if ( @props.project? && 'pan and zoom' in @props.project?.experimental_tools)
+          <div className="pan-zoom-controls" >
+            <div className="draw-pan-toggle" >
+              <div onClick={@togglePan} className={if @state.panEnabled then "" else "active"} >
+                <button title={"draw"} className={"fa fa-mouse-pointer"} title={"annotate"} onClick={@togglePanOff}/>
```

Comment:
I'm not sure why you've wrapped one `onClick` handler in a second `onClick` handler here.